src/config/index.ts

```diff
@@ -285,6 +296,47 @@ export const getRateLimit = () => {
   return config.rateLimit;
 };
 
+export const getMaxPackSizeBytes = (): number => {
+  const config = loadFullConfiguration();
+  const configuredValue = config.limits?.maxPackSizeBytes;
+  const fallback = 1024 * 1024 * 1024; // 1 GiB default
```

Comment:
What are the implications of having a 1 GB default max pack size? Would this somehow prevent larger operations from going through SSH?